frontend/src/patientApp/LanguageToggler.tsx

```diff
@@ -0,0 +1,17 @@
+import Button from "../app/commonComponents/Button/Button";
+import i18n from "../i18n";
+
+export default function LanguageToggler() {
+  return (
+    <Button
+      icon={"globe"}
+      className="usa-button--unstyled"
+      onClick={async () => {
+        const displayLanguage = i18n.language === "en" ? "es" : "en";
+        await i18n.changeLanguage(displayLanguage);
```

Comment:
async/await here isn't necessary since nothing comes after the promise that needs to wait for it